RELEASING.md

```diff
@@ -35,5 +35,30 @@ Failure to do so will leave things in a broken state.
     git push upstream
     ```
 
+## Versioning:
+
+Once tag is created, it's time to use that tag for Runtime Versioning
+
+1. Create a new brach for updating version information in `./sdk/src/version.cc`.
+    ```
+        git checkout -b update_version_${tag} master
+    ```
+2. Run the pre-commit script to update the version:
+    ```console
+        ./buildscripts/pre-commit
+    ```
+
+3. Check if any changes masde since last release broke ABI compatibility. If yes, update `OPENTELEMETRY_ABI_VERSION_NO` in [version.h](./api/include/opentelemetry/version.h).
```

Comment:
```suggestion
3. Check if any changes made since last release broke ABI compatibility. If yes, update `OPENTELEMETRY_ABI_VERSION_NO` in [version.h](./api/include/opentelemetry/version.h).
```